pkg/resources/file_format.go

```diff
@@ -301,7 +301,7 @@ func FileFormat() *schema.Resource {
 		Read:   ReadFileFormat,
 		Update: UpdateFileFormat,
 		Delete: DeleteFileFormat,
-		Exists: FileFormatExists,
+		Check:  CheckFileFormat,
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>unknown field Check in struct literal